src/home_robot/home_robot/navigation_planner/discrete_planner.py

```diff
@@ -486,94 +519,214 @@ def _get_short_term_goal(
         # Create inverse map of obstacles - this is territory we assume is traversible
         # Traversible is now the map
         traversible = 1 - dilated_obstacles
-        traversible[self.collision_map[gx1:gx2, gy1:gy2][x1:x2, y1:y2] == 1] = 0
+        # traversible[self.collision_map[gx1:gx2, gy1:gy2][x1:x2, y1:y2] == 1] = 0
         traversible[self.visited_map[gx1:gx2, gy1:gy2][x1:x2, y1:y2] == 1] = 1
-        agent_rad = self.agent_cell_radius
-        traversible[
-            int(start[0] - x1) - agent_rad : int(start[0] - x1) + agent_rad + 1,
-            int(start[1] - y1) - agent_rad : int(start[1] - y1) + agent_rad + 1,
-        ] = 1
-        traversible = add_boundary(traversible)
-        goal_map = add_boundary(goal_map, value=0)
-        planner = FMMPlanner(
-            traversible,
-            step_size=self.step_size,
-            vis_dir=self.vis_dir,
-            visualize=self.visualize,
-            print_images=self.print_images,
-            goal_tolerance=self.goal_tolerance,
-        )
-        if plan_to_dilated_goal:
-            # Compute dilated goal map for use with simulation code - use this to compute closest goal
-            dilated_goal_map = cv2.dilate(
-                goal_map, self.goal_dilation_selem, iterations=1
-            )
-            # Set multi goal to the dilated goal map
-            # We will now try to find a path to any of these spaces
-            self.dd = planner.set_multi_goal(
-                dilated_goal_map,
-                self.timestep,
-                self.dd,
-                self.map_downsample_factor,
-                self.map_update_frequency,
+        # traversible[
+        # int(start[0] - x1) - agent_rad : int(start[0] - x1) + agent_rad + 1,
+        # int(start[1] - y1) - agent_rad : int(start[1] - y1) + agent_rad + 1,
+        # ] = 1
+        # traversible = add_boundary(traversible)
+        # goal_map = add_boundary(goal_map, value=0)
+        obstacles = 1 - traversible
```

Comment:
As far as I recollect, these planner changes (made for real Spot in GOAT) were breaking in simulation. I will update the config to use old planner for OVMM